lambda/simplif.ml

```diff
@@ -201,19 +208,19 @@ let simplify_exits lam =
   *)
 
   let subst = Hashtbl.create 17 in
-
-  let rec simplif = function
+  
```

Comment:
There are two extra spaces on this line, most likely added by your editor; you should remove them (and keep the empty line).